### PR TITLE
python3Packages.datasette-publish-fly: init at 1.0.2

### PR DESCRIPTION
**Pull request by MostAwesomeDude**
###### Motivation for this change

See #129072, as it's the same sort of plugin.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by risicle** [comment]
Builds happily, macos 10.15 & nixos x86_64

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 136484` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python38Packages.datasette-publish-fly</li>
    <li>python39Packages.datasette-publish-fly</li>
  </ul>
</details>

File: pkgs/development/python-modules/datasette-publish-fly/default.nix [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, datasette
+, flyctl
```
Comment (fabaff) [inline pkgs/development/python-modules/datasette-publish-fly/default.nix:5]:
```suggestion
```

Comment (fabaff) [reply pkgs/development/python-modules/datasette-publish-fly/default.nix:5]: Unused input

**Comment by SuperSandro2000** [comment]
Closing due to inactivity from author.

**Pull request closed without merging**